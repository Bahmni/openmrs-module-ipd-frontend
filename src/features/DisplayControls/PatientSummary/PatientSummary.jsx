import React, { useEffect } from "react";

const PatientSummary = () => {
  useEffect(() => {
    console.log("mounting");

    return () => {
      console.log("unmounting");
    };
  }, []);

  return <div style={{ padding: "10px" }}>PatientSummary</div>;
};

export default PatientSummary;
