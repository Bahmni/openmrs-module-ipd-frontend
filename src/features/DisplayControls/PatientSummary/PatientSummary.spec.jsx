import React from "react";
import { render } from "@testing-library/react";
import PatientSummary from "./PatientSummary";

describe("Patient Summary", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<PatientSummary />);
    expect(asFragment()).toMatchSnapshot();
  });
});
