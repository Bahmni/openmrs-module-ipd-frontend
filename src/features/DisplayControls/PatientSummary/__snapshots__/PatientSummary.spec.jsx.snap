// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Patient Summary should match snapshot 1`] = `
<DocumentFragment>
  <div
    style="padding: 10px;"
  >
    PatientSummary
  </div>
</DocumentFragment>
`;
